src/library/Extensions/V1ContainerExtensions.cs

```diff
@@ -14,7 +14,9 @@ public static bool IsKnownSidecarContainer(this V1Container container)
             return StringComparer.OrdinalIgnoreCase.Equals(container.Name, "devspaces-proxy")
                     || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "istio-proxy")
                     || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "daprd")
-                    || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "jaeger-agent");
+                    || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "jaeger-agent")
+                    || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "linkerd-proxy")
+                    || StringComparer.OrdinalIgnoreCase.Equals(container.Name, "nginx-proxy");

```

Comment:
Maybe slap these in an array sometime so there's less duplicated code? You could iterate over the known sidecar array and perform the check on each.